src/genesis.rs

```diff
@@ -378,6 +358,21 @@ impl Genesis {
             args.push(path);
         }
 
+        if !self.flags.skip_primordial_stakes {
+            for i in 0..num_validators {
+                args.push("--bootstrap-validator".to_string());
```

Comment:
This doesn't work quite right. By marking these as bootstrap validators, they use the `GenesisFlags::bootstrap_validator_sol` and `GenesisFlags::bootstrap_validator_stake_sol` values, instead of `GenesisFlags::internal_node_sol` and `GenesisFlags::internal_node_stake_sol`.

Unless we want to update the `solana-genesis` crate/bin (which isn't totally out of the question, but is a bit of a hassle), I think we'll have to populate an accounts file and use the `--primordial-accounts-file` flag.

Reply:
oh shoot ok ya let's do `--primordial-accounts-file` flag

Reply:
hey been struggling with this a little. in the `net/` scripts, it appears that when validator stakes are baked into genesis it uses the `bootstrap_validator_stake_sol` for all of the validators. But I am having a hard time getting the `--primordial-accounts-file` flag setup properly to work with vote and stake accounts. do you know how to setup these stake/vote accounts in the `primordial-accounts-file` and then get them recognized at the start of the cluster?

Reply:
i could see how we could modify `solana-genesis` as you mentioned to get this to work. but definitely a little more of a hassle. Although just can't quite figure out how to properly add stake and vote accounts to a primordial accounts file. it doesn't appear `net/` scripts support that either. but could be missing it

Reply:
I'll take a look later today!